### PR TITLE
feat: warn if some confused configurations exist in config

### LINKED ISSUE #620: 兼容 config.bodyparser 写法
* **Node Version**: 6.10.0
* **Egg Version**: undefined
* **Plugin Name**: koa-bodyparser
* **Plugin Version**: 2.4.0
* **Platform**: osx

In `config.js`

错误 :

```
exports.bodyparser = {
  formLimit: '1000mb',
  jsonLimit: '1000mb',
  textLimit: '1000mb',
};
```

正确 :

```
exports.bodyParser = {
  formLimit: '1000mb',
  jsonLimit: '1000mb',
  textLimit: '1000mb',
};
```

但是 [koa-bodyparser](https://github.com/koajs/bodyparser) 插件名是小写，能否做兼容？

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

closes #620

### CODE REVIEW

File: config/config.default.js [hunk 1]
```diff
@@ -117,6 +117,22 @@ module.exports = appInfo => {
...
+     * configurations are confused to users
+     * {
+     *   [unexpectedKey]: [expectedKey],
+     * }
+     * @member Config#confusedConfigurations
+     * @type {Object}
+     */
+    confusedConfigurations: {
+      bodyparser: 'bodyParser',
+      notFound: 'notfound',
```
Comment (fengmk2) [inline config/config.default.js]: 命令规则好不一致

Comment (atian25) [inline config/config.default.js]: 确实

Comment (dead-horse) [inline config/config.default.js]: 是的，木已成舟。。

Comment (atian25) [inline config/config.default.js]: 还好吧，统一格式，然后兼容下？

Comment (dead-horse) [inline config/config.default.js]: 不想做兼容，以后插件多了也没法全统一的，就这样吧。

**Pull request merged**